app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-core.js

```diff
@@ -13,6 +13,10 @@ window.GOVUK.analyticsGA4 = window.GOVUK.analyticsGA4 || {};
       'https://www.googletagmanager.com/gtm.js?id='+i+dl+ '&gtm_auth=' + window.GOVUK.analyticsGA4.vars.auth + '&gtm_preview=' + window.GOVUK.analyticsGA4.vars.preview + '&gtm_cookies_win=x';f.parentNode.insertBefore(j,f);
       })(window,document,'script','dataLayer',window.GOVUK.analyticsGA4.vars.id);
       /* eslint-enable */
+    },
+
+    sendData: function (data) {
+      window.dataLayer.push(data)
     }
```

Comment:
Fantastic stuff - this will really help should we need to switch between GTM and GTAG 👏